.github/PULL_REQUEST_TEMPLATE/sniff_change.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Sniff change
+about: Introduce a new sniff or update an existing sniff
+labels: "yoast cs/qa"
+---
+
+## Description
+
+*
+
+
+Refs:
+<!-- Add relevant links to, for instance, related upstream issues. -->
+*
+
+Fixes #
```

Comment:
In all other Yoast repos, this is the final line of the template. Let's go for consistency.

Reply:
Will do, though I still find that illogical in all other repos.

The issue being fixed will normally contain additional information which can enrich the PR description and inform the testing of the PR, so IMHO having it between description and test instructions seems way more in line with flow.